src/core/mutatus.scala

```diff
@@ -27,17 +27,12 @@ import scala.util.{Success, Try}
 /** Mutatus package object */
 object `package` {
 
-  /** provides a default instance of the GCP Datastore service */
-  implicit val defaultDataStore: Service = Service(
-    DatastoreOptions.getDefaultInstance.getService
-  )
-
   /** provides `saveAll` and `deleteAll` methods for collections of case class instances */
   implicit class DataBatchExt[T <: Product](values: Traversable[T]) {
 
     /** saves the all case class as a Datastore entity in batch mode */
     def saveAll()(
-        implicit svc: Service,
+        implicit svc: Service = Service.default,
```

Comment:
Would it be possible to move this default to an implicit defined withing the `Service` companion object? I think that defaults on implicit parameters have been known to be problematic, and defining it in the implicit scope of the companion object should have the same effect.

Reply:
I've changed it in first place due to ambiguous definitions, when I've tried to pass another implicit via method parameters. I've forget that I could just move it to companion object